testapps/setup_testapp_python3_sqlite_openssl.py

```diff
@@ -0,0 +1,30 @@
+
+from distutils.core import setup
+from setuptools import find_packages
+
+options = {'apk': {'requirements': 'libffi,openssl,sqlite3,requests,peewee,sdl2,pyjnius,kivy,python3',
+                   'android-api': 27,
+                   'ndk-api': 21,
+                   'dist-name': 'bdisttest_python3_sqlite_openssl_googlendk',
+                   'ndk-version': '10.3.2',
+                   'arch': 'armeabi-v7a',
+                   'permissions': ['INTERNET', 'VIBRATE'],
+                   }}
+
+package_data = {'': ['*.py',
+                     '*.png']
+                }
+
+packages = find_packages()
+print('packages are', packages)
```

Comment:
out-of-scope-personal-taste: I would drop these two lines above. But yes it mainly comes from copy pasting so I understand it could be a bit out of scope